examples/http-compression/http_compression.bal

```diff
@@ -0,0 +1,98 @@
+import ballerina/http;
+import ballerina/log;
+
+endpoint http:Listener listenerEndpoint {
+    port: 9090
+};
+
+// Since compression behaviour of the service is set as COMPRESSION_AUTO, entity body compression is done according
```

Comment:
Let's use single backticks for "COMPRESSION_AUTO". Apply elsewhere for similar settings that you might do.